src/components/tutor-input.cjsx

```diff
@@ -47,15 +48,12 @@ TutorInput = React.createClass
   forwardLabelClick: -> @focus()
 
   render: ->
-    classes = ['form-control']
-    wrapperClasses = ["form-control-wrapper", "tutor-input"]
-    wrapperClasses.push(@props.className) if @props.className
+    classes = classnames 'form-control', @props.class,
```

Comment:
No, but I suspect that it probably should been.  There's a  `classes.push(@props.class)` on the old line 58 below.  I opted to leave it as-is for now.